src/java/org/apache/cassandra/schema/SchemaConstants.java

```diff
@@ -41,6 +41,8 @@
 public final class SchemaConstants
 {
     public static final Pattern PATTERN_WORD_CHARS = Pattern.compile("\\w+");
+    public static final Pattern PATTERN_NON_WORD_CHAR = Pattern.compile("\\W");
+
```

Comment:
@k-rus remove this redundant empty line please

Reply:
Fixed